### PR TITLE
Make requests-toolbelt and progressbar a Depends.

### PR DESCRIPTION
**Pull request by kyrofa**
This PR fixes LP: [#1541349](1541349), where if Snapcraft is installed via the .deb and a user tries to login/upload a package, they get modules not found.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -32,8 +32,10 @@ Depends: python3-apt,
          python3-jsonschema,
          python3-lxml,
          python3-pkg-resources,
+         python3-progressbar
```
Comment (sergiusens) [inline debian/control]: missing `,`

File: debian/control [hunk 2]
```diff
@@ -32,8 +32,10 @@ Depends: python3-apt,
...
          python3-requests,
          python3-requests-oauthlib,
+         python3-requests-toolbelt
```
Comment (sergiusens) [inline debian/control]: missing `,`

**Comment by sergiusens** [comment]
:+1: but I want @dholbach to also ack :wink:

**Comment by dholbach** [comment]
:+1: - go go go! :smile:

**Pull request merged**